build.gradle.kts

```
@@ -0,0 +1,43 @@
+plugins {
+    id("com.hivemq.extension")
+    id("org.asciidoctor.jvm.convert")
+}
+
+group = "com.hivemq"
+description = "Prometheus Monitoring Extension"
+
+hivemqExtension {
+    name = "Prometheus Monitoring Extension"
+    author = "HiveMQ"
+    priority = 0
```

```suggestion
    priority = 1000
```
Should not be changed, before it was 1000

---

done